src/internal/api/packages/find.go

```diff
@@ -38,7 +39,20 @@ func findPackage(pattern *regexp.Regexp, w http.ResponseWriter, setDir func() (s
 		message.ErrorWebf(err, w, "Error getting directory")
 	}
 
+	// TODO: decide as a team how JSON errors should be formatted
+	type JSONError struct {
```

Comment:
Agreed that the todo should stay, but can you move this to `internal/api/common/common.go`? There is some other response writing stuff there already.